script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9B.txt

```diff
@@ -37,15 +37,15 @@ Having a girl around my own age look after me isn't to my taste right now.
 Hisui bows deeply.
 The prickling pain in my chest is brought on by my selfish guilt.
 <Page11>
-"All right. From the way Akiha puts it, she's not going to let me be alone. Besides, I don't think making her dismiss you just to make allowance for me is a good thing. So I'll accept your help but―――"//「分かった。秋葉のあの言いぶりじゃ俺を一人にはしないだろうし、そもそも俺の気持ちの問題で君を解雇させるのは良くはないと思う。ありがたくお世話になるけど―――」
+"All right. From the way Akiha puts it, she's not going to let me be alone. Besides, I don't think making her dismiss you just to make allowance for me is a good thing. So I'll accept your help but―――"//「分かった。秋葉のあの言いぶりじゃ俺を一人にはしないだろうし、そもそも俺の気持ちの問題で君を解雇させるのは良くはないと思う。ありがたくお世話になるけど―――」, what does Shiki mean by allowance here? maybe change the word allowance?
```

Comment:
Make allowances for, accommodate his preferences (not having a servant).

Reply:
> I don't think making her dismiss you just to accommodate me is a good thing

That actually reads better to me

Reply:
I'm good with `accommodate`